### ISSUE TITLE
Edit location or delete and create another one

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#9125]
> What application are you using?

Data Tracker

> Describe the problem.

Is there any way I can edit this location or should it be deleted and another one created?
The ATD Location ID is:  61f7f8dce6ded9f7cacac5f7

> How soon do you need this?

Soon — This week 

> Requested By
Renee O.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/627d36c7136f140020ac540d/screenshot20220512113247.jpg) (78.33kb)

Request ID: DTS22-103975

**Comment by dianamartin** [comment]
Looked into this location to find the actual Location ID. I was able to find it on the Locations table: [LOC22-021720](https://atd.knack.com/amd#locations/view-location-details/61f7f8dce6ded9f7cacac5f7/). 

Emailed user 5/12
>Got your request about the edit/delete location for: [LOC22-021720], wanted to know when you'd be available to sync on it real quick

**Comment by dianamartin** [comment]
Synced with Renee 5/12
We found the location she was looking for in the Locations list, she didn't see it the first time she looked. We made a discovery that the Locations map used to create a new location isn't the normal AMD Asset Map. It's called "ATD AMD Signal Request Evaluation Assistant" web map. It has a layer that is the "atd_arterial_management_locations" that shows up in the pop-up but it doesn't show much information, it looks like this:
![image](https://user-images.githubusercontent.com/21112975/168149753-8a89e1cb-222b-41be-9975-f66f62a397c1.png)

She was hoping to add the "Location ID" that can be searched in Data Tracker. This AGOL layer is updated via script so, I we'd need to do some discovery as to why the other fields aren't populating on the web layer.
